src/NuGet.Clients/NuGet.PackageManagement.UI/DisplayVersion.cs

```diff
@@ -48,22 +51,28 @@ public class DisplayVersion
             Version = range.MinVersion;
             IsCurrentInstalled = isCurrentInstalled;
             AutoReferenced = autoReferenced;
+            IsDeprecated = isDeprecated;
 
             // Display a single version if the range is locked
-            if (range.HasLowerAndUpperBounds && range.MinVersion == range.MaxVersion)
-            {
-                var formattedVersionString = Version.ToString(versionFormat, VersionFormatter.Instance);
+            // If range is unlocked, display it and use the original value for floating ranges
+            var versionString = range.HasLowerAndUpperBounds && range.MinVersion == range.MaxVersion ?
+                Version.ToString(versionFormat, VersionFormatter.Instance)
+                : Range.OriginalString;
 
-                _toString = string.IsNullOrEmpty(AdditionalInfo) ?
-                    formattedVersionString :
-                    AdditionalInfo + " " + formattedVersionString;
+            _toString = "";
+            if (!string.IsNullOrEmpty(AdditionalInfo))
+            {
+                _toString += AdditionalInfo + " ";
             }
-            else
+
+            _toString += versionString;
+
+            if (IsDeprecated)
             {
-                // Display the range, use the original value for floating ranges
-                _toString = string.IsNullOrEmpty(AdditionalInfo) ?
-                    Range.OriginalString :
-                    AdditionalInfo + " " + Range.OriginalString;
+                _toString += string.Format(
```

Comment:
Avoid refactoring all of the above, if you're only gonna append Deprecated to the string. 

We need to mindful of unnecessary changes this late in the game.

Reply:
good call

Reply:
note that the refactoring above was removed but this comment was on a line that didn't change